### PR TITLE
TerminalImages for custom items added + custom shop item info added in customscrap

### PR DESCRIPTION
**Pull request by XuuXiao**

### CODE REVIEW

File: docs/dev/apis/lethallib/custom-enemies/unity-project.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ prev: true
 next: true
```
Comment (Xilophor) [inline docs/dev/apis/lethallib/custom-enemies/unity-project.md:3]: Please make sure that your branch is up-to-date with main; there seems to be some GitHub shenanigans going on with this file. Had a moment of deja-vu, until I realized I indeed saw this with the other PR.

File: docs/dev/apis/lethallib/customscrap.md [hunk 2]
```diff
@@ -83,6 +106,39 @@ The ScanNode object will also need to have the layer set to "ScanNode".
 
 After you've done this, you should go back to your Item Data and assign the Spawn Prefab to be the item prefab.
 
+## TerminalNode
+To make a buy-able **Shop Item**, you must make a custom `TerminalNode` for your item, I initially found this confusing as it needed a few `TerminalNodes` and a `TerminalKeyword` to get it working.
+1. Create 3 `TerminalNode` `ScriptableObject`'s and 2 `TerminalKeyword` `ScriptableObject`'s, Usually you'd only need 2 `TerminalNode`'s, but for the first setup, you'll need all 5.
+2. Name the `TerminalKeyword`'s "Confirm" and "Deny" (This is for when you're confirming or denying buying the item).
+3. Name one of the `TerminalNode`'s "CancelPurchase" (This will be reusable for future shop items).
+4. Keep the following `TerminalKeyword`'s and `TerminalNode` to be as close to the screenshots as possible, this is the vanilla standard that's used for everytime (hence can be repeatedly used).
+![Screenshot: Deny TerminalKeyword](/images/lethallib/custom-scrap/DenyTerminalKeyword.png)
```
Comment (Xilophor) [inline docs/dev/apis/lethallib/customscrap.md:115]: Make sure the link directory matches the file directory (`custom-scrap` vs `customscrap`) - causes build error.

**Review by Xilophor — CHANGES_REQUESTED** [review]

File: docs/dev/apis/lethallib/customscrap.md (hunk 2 above)
Comment (XuuXiao) [reply docs/dev/apis/lethallib/customscrap.md]: this is done now.

File: docs/dev/apis/lethallib/custom-enemies/unity-project.md (hunk 1 above)
Comment (XuuXiao) [reply docs/dev/apis/lethallib/custom-enemies/unity-project.md]: I think i got it all sorted now, please check

**Review by Xilophor — APPROVED** [review]
lgtm!

**Pull request merged**